test/clojure/core_test/doseq.cljc

```diff
@@ -0,0 +1,68 @@
+(ns clojure.core-test.doseq
+  (:require [clojure.test :as t :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer) [when-var-exists]]))
+
+
+(when-var-exists doseq
+  (deftest returns-nil-and-basic-iteration
```

Comment:
Please have only one `deftest`, called `test-doseq`. Everything within that should be various `testing` forms.